File: Java/AD - HOC/ 1026 - Carrega ou não Carrega.java
import java.util.Scanner;

/*
 * Autor: Pierre Vieira
 * Data de submissão: 26/01/2020 15:20:36
 */
public class Main {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        long x, y;
        while (teclado.hasNext()) {
            x = teclado.nextLong();
            y = teclado.nextLong();
            System.out.println(x ^ y);
        }
    }
}

File: C/AD - HOC/1026 - Carrega ou não Carrega.c
/*
 * Autor: Pierre Vieira
 * Data de submissão: 26/01/2020 15:01:03
 */

#include <stdio.h>

int main(){
    long long x, y;
    while (scanf("%lld %lld", &x, &y) != EOF){
        printf("%lld\n", x^y);
    }
    return 0;
}